tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932175.yaml

```diff
@@ -0,0 +1,113 @@
+---
+meta:
+  author: "Felipe Zipitria"
+  description: "Remote Command Execution: shell aliasing detection"
+  enabled: true
+  name: 932175.yaml
+tests:
+  - test_title: 932175-1
+    desc: "Test for 0FA9HCGT alias eennvv=env&eennvv whoami"
+    stages:
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            headers:
+              Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
+              Host: localhost
+              User-Agent: OWASP ModSecurity Core Rule Set
+            method: GET
+            port: 80
+            uri: /get?rce=alias%20eennvv%3Denv%26eennvv%20whoami
+            version: HTTP/1.1
+          output:
+            log_contains: id "932175"
+  - test_title: 932175-2
+    desc: "Test for alias with shell tricks"
+    stages:
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            headers:
+              Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
+              Host: localhost
+              User-Agent: OWASP ModSecurity Core Rule Set
+            method: POST
+            port: 80
+            uri: /post
+            data: |
+              rce=alias a=c${KK}url&a google.com
+            version: HTTP/1.1
+          output:
+            log_contains: id "932175"
+  - test_title: 932175-3
+    desc: "Test for alias with single quoting"
+    stages:
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            headers:
+              Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
+              Host: localhost
+              User-Agent: OWASP ModSecurity Core Rule Set
+            method: POST
+            port: 80
+            uri: /post
+            data: |
+              rce=alias 'letter'=curl&letter google.com
+            version: HTTP/1.1
+          output:
+            log_contains: id "932175"
+  - test_title: 932175-4
```

Comment:
Can you add two more tests to match quotes in the alias name, e.g., `curl='quote'`?

Reply:
Sure!